packages/bentley/grouping-mapping-widget/src/widget/icons/DatabaseInfo.svg

```diff
@@ -0,0 +1,6 @@
+<svg version="1.1" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">

```

Comment:
was this svg not in itwin/itwinui-icons-react?

Reply:
No it is a custom made SVG.